### PR TITLE
 [Bug fixed]: Leaderboard badges malfunctioning in search results

### LINKED ISSUE #392: [Bug]: Leaderboard badges malfunctioning in search results
### Issue
https://github.com/ashutosh-rath02/git-re/pull/376#issuecomment-2231411732

Please assign this issue to me.

### PR DESCRIPTION
**Pull request by dhairyathedev**
Fixes #392

### CODE REVIEW

**Comment by ashutosh-rath02** [comment]
Thank you for submitting your pull request! 🙌 We'll review it as soon as possible. In the meantime, please ensure that your changes align with our [CONTRIBUTING.md](https://github.com/ashutosh-rath02/git-re/blob/main/CONTRIBUTING.md). If there are any specific instructions or feedback regarding your PR, we'll provide them here. Thanks again for your contribution! 😊

File: app/leaderboard/page.tsx [hunk 1]
```diff
@@ -108,51 +113,33 @@ export default function Leaderboard() {
...
+                      {user.username}
+                    </Link>
                   </TableCell>
+                  <TableCell>{user.rating}</TableCell>
                 </TableRow>
               ))}
             </TableBody>
+          ) : (
+            <TableBody>
+              {/* ... (skeleton loading code remains the same) ... */}
```
Comment (ashutosh-rath02) [inline app/leaderboard/page.tsx:141]: What's this?

**Review by ashutosh-rath02 — CHANGES_REQUESTED** [review]
Complete the code

File: app/leaderboard/page.tsx (hunk 1 above)
Comment (dhairyathedev) [reply app/leaderboard/page.tsx]: Resolved @ashutosh-rath02

**Review by ashutosh-rath02 — APPROVED** [review]
LGTM 🥊

**Pull request merged**